text-ui-test/duke.json

```diff
@@ -0,0 +1,41 @@
+{
+  "taskList": [
+    {
+      "type": "todo",
+      "task": " Buy milk",
+      "isDone": false
+    },
+    {
+      "dueBy": " 17/02/2023",
+      "type": "Deadline",
+      "task": "bake a cake ",
+      "isDone": true
+    },
+    {
+      "startTime": " 16/02/2023 ",
+      "endTime": " 19/02/2023",
+      "type": "Event",
+      "task": "attend baking course ",
+      "isDone": true
```

Comment:
Understand that you are using a json file to store your variables. Do check w the prof if it can be used.

Reply:
Hi, I think prof approved the gson library in the forum https://github.com/nus-cs2113-AY2223S2/forum/issues/22, should be fine right?

Reply:
Ah! Should be fine then !good work !